src/dolphin/ipc.ts

```diff
@@ -35,7 +35,7 @@ export const ipc_launchNetplayDolphin = makeEndpoint.main("launchNetplayDolphin"
 
 export const ipc_migrateDolphin = makeEndpoint.main(
   "migrateDolphin",
-  <{ migrateNetplay: string | null; migratePlayback: boolean }>_,
+  <{ migrateNetplay: string | null; migratePlayback: boolean; migratePlaybackPath?: string }>_,
   <SuccessPayload>_,
 );
```

Comment:
While I guess this is not the worst, do you think we can generalize this IPC endpoint over the two Dolphin types?

For example, maybe something like this:

1. `ipc_importDolphinSettings`: IPC endpoint which accepts a path string and a dolphin type (either netplay or dolphin) and just copies the folder over (and doesn't delete anything).

2. `ipc_completeDolphinMigrationStep`: IPC endpoint which accepts no parameters, but simply "completes the migration step" by deleting the old slippi desktop app folder.

I guess this does mean that for the migration step, the renderer will need to know the location of the old desktop app path, but for the purpose of the migration, not for the purpose of the completion (the folder deletion). The end of the migration quick start step, simply calls calls `complete` to remove the previous path.

What are your thoughts on something like this?